tikv-control.md

```diff
@@ -310,18 +310,21 @@ Use the `compact` command to manually compact data of each TiKV.
 - To compact data in the local mode, use the following command:
 
     ```shell
-    tikv-ctl --data-dir /path/to/tikv compact --db kv
+    tikv-ctl --data-dir /path/to/tikv compact --d kv
     ```
 
 - To compact data in the remote mode, use the following command:
 
     ```shell
-    tikv-ctl --host ip:port compact --db kv
+    tikv-ctl --host ip:port compact --d kv
     ```
 
 ### Compact data of the whole TiKV cluster manually
 
-Use the `compact-cluster` command to manually compact data of the whole TiKV cluster. The flags of this command have the same meanings and usage as those of the `compact` command.
+Use the `compact-cluster` command to manually compact data of the whole TiKV cluster. The flags of this command have the same meanings and usage as those of the `compact` command. The only difference is as follows:
+
+- For the `compact-cluster` command, you use `--pd` to specify the address of the PD as the compact target.
```

Comment:
```suggestion
- For the `compact-cluster` command, you use `--pd` to specify the address of the PD so `tikv-ctl` will be able to find all TiKVs in the cluster as compact target.
```

I think the target of compaction should be TiKV nodes, PDs are just metadata provider.